01-Git_Bash_Make/guess_a_number/Makefile

```diff
@@ -0,0 +1,12 @@
+CC=clang
+CC_FLAGS=-O3 -target x86_64-mingw64
+
+guess_a_number.exe: obj
+	${CC} *.o -o $@ ${CC_FLAGS}
+
+obj: main.c lib.c
+	${CC} $^ -c ${CC_FLAGS}
+
+clean:
+	del *.o
+	del *.exe
```

Comment:
Please add empty line at the end of makefile

Reply:
[Done](https://github.com/Kernel-GL-HRK/gl-kernel-basecamp-2019/pull/77/commits/7ab853cd4976e51787ad5ccfd50ad8976c1ef6eb).